Makefile

```diff
@@ -28,9 +28,7 @@ test:
 	@tox
 
 testdeps:
-	pip install tox
-	pip install tox-pip-version
-	pip install tox-setuptools-version
+	. tools/setup_pyenv.sh
```

Comment:
I thought it was a good idea to call the script from the Makefile, but I have no idea if this is a good practice.
Should I instead copy the script contents and maintain it here?

Reply:
I think it is okay to keep the script, but I think @paride will not that better than me

Reply:
Per standup discussion, I think we came to the conclusion to make the script more instructional than auto-configuring, and to document its usage as a devel environment setup step rather than running in as part of the `testdeps` target.

Reply:
done